dnf-plugins-extras.spec

```diff
@@ -32,6 +32,7 @@ Extras Plugins for DNF.
 
 %package -n %{name}-common-data
 Summary:        Common data files for Extras Plugins for DNF
+Provides:       %{name} = %{version}-%{release}
```

Comment:
that doesn't make any sense. when people do `dnf install dnf-plugins-extras`, it probably should do something else than installing man pages ;)